### ISSUE TITLE
npm version

### ISSUE THREAD

**Issue by kubenstein** [gabrielcsapo/node-git-server#4]
hi,
first of all thank you for that project, thanks to you i was able to build a MVP of my project nearly just over a weekend. Unfortunately `node-git-server` npm version is outdated compared to github. Can you please release current github code as an npm package? It will make my dependencies more stable.

Thank you!

**Comment by gabrielcsapo** [comment]
@kubenstein just thought about this package today, I will close out this as well as issue #2

**Comment by gabrielcsapo** [comment]
@kubenstein just published 

```
node-git-server gabrielcsapo$ npm publish .
+ node-git-server@0.0.1
```

can you tell me if this works (https://github.com/gabrielcsapo/node-git-server/commit/6a058557b3eda33761de232c72f92741d15f2b00)?

**Comment by kubenstein** [comment]
hi,
i checked the npm version and things are working so thank you for a quick response and push.

i also spotted a bug (and found a fix already) so please take a look at #2 

Thank you!

**Comment by gabrielcsapo** [comment]
@kubenstein published as `0.0.2` (https://github.com/gabrielcsapo/node-git-server/commit/97063d98d0e49b92001d71580ec5c29fd5fcdede)
